boards/shields/reel_board_canbus/reel_board_canbus.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_SPI=y
+CONFIG_SPI_1=y
+CONFIG_CAN_MCP2515=y
```

Comment:
neither CAN_1 nor CAN_0 is enabled, device label is "CAN_1", should CONFIG_CAN_1=y be here?

Reply:
Yes. Had to delete it in the proj.conf and forgot to add it here.